pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/ContextImpl.java

```diff
@@ -82,6 +84,10 @@
  */
 @ToString(exclude = {"pulsarAdmin"})
 class ContextImpl implements Context, SinkContext, SourceContext, AutoCloseable {
+    private static final String CUSTOM_METRIC_PREFIX = "custom_metric_";
```

Comment:
what would the new exposed metrics name be? `pulsar_function_custom_metric_`? And then the metric name is a label like before?

Reply:
For a metric named `metric_demo_counter` it would be `pulsar_source_custom_metric_demo_counter`. And the labels would be directed added to the metrics (see example above)